### PR TITLE
Fix file path for counted jobs

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: backend/faust/job_worker.py [hunk 1]
```diff
@@ -490,6 +497,22 @@ async def read_slurm_out(slurm_id: int, workdir: str) -> Union[str, bytes, None]
 completed_jobs = set()
 
 
+def _scan_path(job_type: JobType, scan: Scan) -> str:
+    date_dir = scan.created.astimezone().strftime(DATE_DIR_FORMAT)
+    path = AsyncPath(settings.JOB_NCEMHUB_RAW_DATA_PATH) / date_dir
```
Comment (swelborn) [inline backend/faust/job_worker.py:502]: I would probably put this into an `else` below `JobType.COUNT` for clarity.

**Review by swelborn — APPROVED** [review]
lgtm, take or leave my comment inline

File: backend/faust/job_worker.py (hunk 1 above)
Comment (cjh1) [reply backend/faust/job_worker.py]: Fair point, done

**Comment by cjh1** [comment]
@swelborn Can you review the build changes?

**Review by swelborn — APPROVED** [review]
tested build on both `linux/amd64` and `linux/arm64` and it works.

**Pull request merged**